anet-dictionary.yml

```
@@ -777,6 +777,12 @@ fields:
       placeholder: Fill in the name of this location
     type:
       label: Type
+    digram:
+      label: Digram
+      placeholder: Fill in the 2-letter code for this country
+    trigram:
+      label: Trigram
+      placeholder: Fill in the 3-letter code for this country
```

I propose to follow the ISO 3166 / STANAG 1059 terminology for the default label.
```suggestion
    digram:
      label: Alpha-2 code
      placeholder: Fill in the 2-letter code for this country
    trigram:
      label: Alpha-3 code
      placeholder: Fill in the 3-letter code for this country
```